From f04a247e61d850a483cd9e8587c0e06e778b0893 Mon Sep 17 00:00:00 2001
From: Cyril Chemparathy <cyril@ti.com>
Date: Tue, 4 Dec 2012 18:59:56 -0500
Subject: [PATCH 041/257] dma: keystone: use of_get_child_by_name()

This patch modifies the keystone pktdma driver to use of_get_child_by_name()
available in upstream kernels.
(cherry picked from commit e5f229c1c13752d8c9e60bbe91479369061a2ada)

Signed-off-by: Zumeng Chen <zumeng.chen@windriver.com>
---
 drivers/dma/keystone-pktdma.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/dma/keystone-pktdma.c b/drivers/dma/keystone-pktdma.c
index 11ea6ba..d993e44 100644
--- a/drivers/dma/keystone-pktdma.c
+++ b/drivers/dma/keystone-pktdma.c
@@ -2064,7 +2064,7 @@ static int keystone_dma_probe(struct platform_device *pdev)
 
 	INIT_LIST_HEAD(&engine->channels);
 
-	chans = of_find_child_by_name(node, "channels");
+	chans = of_get_child_by_name(node, "channels");
 	if (!chans) {
 		dev_err(dma_dev(dma), "could not find channels\n");
 		return -ENODEV;
-- 
2.7.4

